docs/en/observability/logs-parse-filter.asciidoc

```diff
@@ -0,0 +1,1227 @@
+* <<logs-stream-reroute>> – route data from the generic data stream to a target data stream
+* <<logs-filter>> – Filter your data to focus on logs that meet specific criteria
+* <<logs-aggregate>> – Analyze and summarize your log data to find patterns and gain insight.
+
+[discrete]
+[[logs-stream-parse]]
+= Parse your logs
+
+Make your logs more useful by extracting structured fields from your unstructured log data. Extracting structured fields makes it easier to search, analyze, and filter your log data. 
+
+Let's look at this log example:
+
+[source,log]
+----
+2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%.
+----
+
+From the *Dev Tools* console found in {ecloud} in the navigation menu under *Management*, add the example log to {es} using the following command:
+
+[source,console]
+----
+POST logs-example-default/_doc
+{
+  "message": "2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%."
+}
+----
+
+The previous command stores the document in the `logs-example-default` data stream. Retrieve it with the following search:
+
+[source,console]
+----
+GET /logs-example-default/_search
+----
+
+The results should look like this:
+
+[source,JSON]
+----
+{
+  ...
+  "hits": {
+    ...
+    "hits": [
+      {
+        "_index": ".ds-logs-example-default-2023.08.09-000001",
+        ...
+        "_source": {
+          "message": "2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%.",
+          "@timestamp": "2023-08-09T17:19:27.73312243Z"
+        }
+      }
+    ]
+  }
+}
+----
+
+{es} indexes the `message` field by default meaning you can search for phrases like `WARN` or `Disk usage exceeds`. For example, the following command searches for the phrase `WARN` in the log `message` field:
+
+[source,console]
+----
+GET logs-example-default/_search
+{
+  "query": {
+    "match": {
+      "message": {
+        "query": "WARN"
+      }
+    }
+  }
+}
+----
+
+While you can search for phrases in the `message` field, you can't use this field to filter log data. Your message, however, contains all of the following potential fields you can extract and use to filter and aggregate your log data:
+
+- *@timestamp* – `2023-08-08T13:45:12.123Z` – Extracting this field lets you sort logs by date and time. This is helpful when you want to view your logs in the order that they occurred or identify when issues happened.
+- *log.level* – `WARN` – Extracting this field lets you filter logs by severity. This is helpful if you want to focus on high-severity WARN or ERROR-level logs, and reduce noise by filtering out low-severity INFO-level logs.
+- *host.ip* – `192.168.1.101` – Extracting this field lets you filter logs by the host IP addresses. This is helpful if you want to focus on specific hosts that you’re having issues with or if you want to find disparities between hosts.
+- *message* – `Disk usage exceeds 90%.` – You can search for keywords in the message field.
```

Comment:
Nit: I stumbled over `keywords` in message.  It is kind of accurate it just confused me at first as keyword is also a type.